index.html

```diff
@@ -0,0 +1,456 @@
+
+        @keyframes slideIn {
+            from {
+                opacity: 0;
+                transform: translateX(-20px);
+            }
+            to {
+                opacity: 1;
+                transform: translateX(0);
+            }
+        }
+
+        .feature-list li:hover {
+            background: rgba(255, 255, 255, 0.08);
+            border-left-color: #667eea;
+            transform: translateX(10px);
+        }
+
+        .controls {
+            display: inline-block;
+            background: rgba(0, 0, 0, 0.3);
+            padding: 8px 16px;
+            border-radius: 8px;
+            font-family: 'Courier New', monospace;
+            font-weight: bold;
+            letter-spacing: 2px;
+        }
+
+        @media (max-width: 768px) {
+            .title {
+                font-size: 2.5rem;
+            }
+
+            .glass-card {
+                padding: 25px;
+            }
+
+            .demo-links {
+                flex-direction: column;
+            }
+
+            .demo-link {
+                min-width: 100%;
+            }
+        }
+
+        .floating-shapes {
+            position: fixed;
+            top: 0;
+            left: 0;
+            width: 100%;
+            height: 100%;
+            pointer-events: none;
+            z-index: 0;
+        }
+
+        .shape {
+            position: absolute;
+            background: rgba(106, 17, 203, 0.1);
+            border-radius: 50%;
+            animation: float 20s infinite ease-in-out;
+        }
+
+        .shape:nth-child(1) {
+            width: 80px;
+            height: 80px;
+            top: 20%;
+            left: 10%;
+            animation-delay: 0s;
+        }
+
+        .shape:nth-child(2) {
+            width: 60px;
+            height: 60px;
+            top: 60%;
+            left: 80%;
+            animation-delay: 5s;
+        }
+
+        .shape:nth-child(3) {
+            width: 100px;
+            height: 100px;
+            top: 80%;
+            left: 20%;
+            animation-delay: 10s;
+        }
+
+        @keyframes float {
+            0%, 100% {
+                transform: translateY(0) rotate(0deg);
+                opacity: 0.3;
+            }
+            50% {
+                transform: translateY(-50px) rotate(180deg);
+                opacity: 0.6;
+            }
+        }
+    </style>
+</head>
+<body>
+    <div class="floating-shapes">
+        <div class="shape"></div>
+        <div class="shape"></div>
+        <div class="shape"></div>
+    </div>
+
+    <div class="container">
+        <div class="header">
+            <h1 class="title">minecraft-selfie</h1>
+            <p class="subtitle">face-controlled minecraft avatars</p>
+        </div>
+
+        <div class="glass-card">
+            <h2 class="section-title">Demos</h2>
+            <p class="info-text">Still very early days, but you can see demos of...</p>
+            <div class="demo-links">
+                <div class="demo-link">
```

Comment:
Are these supposed to be links to the demo pages?

The styling works so they change behaviour on hover but they aren't clickable through to the actual demo pages.